modules/kubefed-document-attributes.adoc

```diff
@@ -0,0 +1,26 @@
+// Standard document attributes to be used in the documentation
+//
+// The following are shared by all documents:
+:toc:
+:toclevels: 4
+:experimental:
+//
+// Product content attributes, that is, substitution variables in the files.
+//
+:product-title: OpenShift Container Platform
```

Comment:
If you're planning on including the "include::modules/common-attributes.adoc[]" in all of your assemblies (which it looks like the assembly in this PR does), I wouldn't redefine product-title in this attribute file too.